integration-test/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,7 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.7-bin.zip
```

Comment:
We can use Gradle 8.10 according versions mapping https://kotlinlang.org/docs/gradle-configure-project.html#apply-the-plugin

Reply:
I choose 8.7 because it is the same version used in the 'big' kotlinx.serialization project